hibernate-core/src/test/java/org/hibernate/test/annotations/duplicatedgenerator/DuplicateTest.java

```diff
@@ -31,13 +32,14 @@ public void testDuplicateEntityName() throws Exception {
 		try {
 			cfg.addAnnotatedClass( Flight.class );
 			cfg.addAnnotatedClass( org.hibernate.test.annotations.Flight.class );
+			cfg.addAnnotatedClass( org.hibernate.test.annotations.Company.class );
 			cfg.addResource( "org/hibernate/test/annotations/orm.xml" );
 			cfg.addResource( "org/hibernate/test/annotations/duplicatedgenerator/orm.xml" );
 			serviceRegistry = ServiceRegistryBuilder.buildServiceRegistry( cfg.getProperties() );
 			sf = cfg.buildSessionFactory( serviceRegistry );
             Assert.fail( "Should not be able to map the same entity name twice" );
 		}
-		catch (AnnotationException ae) {
+		catch (DuplicateMappingException ae) {
```

Comment:
So we had an exception before if the class names were conflicting? How comes the OP didn't have an error?

Reply:
Nope. This test was actually failing because of a missing class. It was useless.